diff --git a/node_modules/react-native-24h-timepicker/index.d.ts b/node_modules/react-native-24h-timepicker/index.d.ts
index 8b21fef..99620a5 100644
--- a/node_modules/react-native-24h-timepicker/index.d.ts
+++ b/node_modules/react-native-24h-timepicker/index.d.ts
@@ -15,7 +15,7 @@ declare module "react-native-24h-timepicker" {
     textCancel?: string;
     textConfirm?: string;
     onCancel?: () => void;
-    onConfirm?: () => void;
+    onConfirm?: (h: string, m: string) => void;
   };
 
   class TimePicker extends Component<TimePickerProps> {
